src/module/service.cpp

```diff
@@ -0,0 +1,245 @@
+#define BOOST_LOG_DYN_LINK 1
```

Comment:
The file above this, src/module/module.cpp is being added as an empty file. Is that necessary for compiling or something? Or just an accidental add?

Reply:
Accidental add, thanks! Will remove.